tests/contrib/django/test_tracing_disabled.py

```diff
@@ -0,0 +1,34 @@
+# 3rd party
+from django.apps import apps
+from django.test import TestCase, override_settings
+
+# project
+from ddtrace.tracer import Tracer
+from ddtrace.contrib.django.conf import settings
+
+# testing
+from ...test_tracer import DummyWriter
+
+
+class DjangoTracingDisabledTest(TestCase):
+    def test_nothing_is_written(self):
+        tracer = Tracer()
+        tracer.writer = DummyWriter()
+        # Backup the old conf
+        backupTracer = settings.TRACER
+        backupEnabled = settings.ENABLED
+        # Disable tracing
+        settings.ENABLED = False
+        settings.TRACER = tracer
+        # Restart the app
+        app = apps.get_app_config('datadog_django')
+        app.ready()
+
+        traces = tracer.writer.pop_traces()
```

Comment:
since the test is `nothing_is_written`, you can add a `tracer.trace()` call to create and finish a `Span`. Calling `app.ready()` will only send services and not traces, so adding something could be great. If you prefer, you can split the test in two: one for services and one for traces.